app/routes/projects/$projectId/edit.tsx

```diff
@@ -149,7 +172,7 @@ export default function EditProjectPage() {
           </Box>
 
           <TabPanel value={tabIndex} index={0}>
-            <ValidatedForm
+            {/* <ValidatedForm
```

Comment:
Should we leave all this code commented? If it will be useful for future work then awesome, but I don't know why we are commenting it, sorry hahaha